media-sound/mpg321/mpg321-0.3.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A simple and lightweight command line MP3 Player"
+DESCRIPTION="mpg321 is a very popular command-line mp3 player. mpg321 is used \
+for frontends, as an mp3 player and as an mp3 to wave file decoder (primarily \
+for use with CD-recording software)."
+HOMEPAGE="http://mpg321.sourceforge.net/"
+COPYRIGHT="2001 Joe Drew
+		2006-2012 Nanakos Chrysostomos"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://ncu.dl.sourceforge.net/project/mpg321/mpg321/$portVersion/mpg321_$portVersion.orig.tar.gz"
```

Comment:
Can you please change SOURCE_URI to:
```
SOURCE_URI="https://downloads.sourceforge.net/mpg321/mpg321_$portVersion.orig.tar.gz"
```
(This is to avoid the use of a specific SourceForge mirror.)